gromacs/testing/nosetester.py

```diff
@@ -0,0 +1,218 @@
+"""
+Nose test running.
```

Comment:
Do we need this or can't we just import `nose`? 

No problem to depend on `nose`.

Also, is this your code? Otherwise we need license and author.